cmd/driver/config/config.go

```diff
@@ -158,7 +158,10 @@ func (o *driverConfigOptions) RunDriverConfig(ctx context.Context, cmd *cobra.Co
 
 			d, err := driverdistro.Discover(info, driverCfg.HostRoot)
 			if err != nil {
-				return err
+				if !errors.Is(err, driverdistro.ErrUnsupported) {
```

Comment:
When error is ErrUnsupported, just print that we detected an unsupported OS (we failed to determine it), and go on using the fallback generic logic.